src/components/EventDetails/EventDetails.js

```diff
@@ -108,16 +120,28 @@ class EventDetails extends Component {
             {loc && renderAddress(loc)}
             {startDate && renderTimeRange(startDate, endDate)}
             <div className={styles.socialBtns}>
-              <a href={shareUrl} className={styles.facebookBtn}>
-                <FaFacebookSquare size={25} />
-                <span>Share on Facebook</span>
-              </a>
-              <a href={shareUrl} className={styles.twitterBtn}>
-                <FaTwitter size={25} />
-                <span>Share on Twitter</span>
-              </a>
+              <button
+                onClick={this.showHideSocialBtns}>
```

Comment:
I haven't pulled the branch to check, but doesn't this need a `.bind(this)` with it?